src/App.tsx

```diff
@@ -1,25 +1,26 @@
+/* eslint-disable no-console */
 import React, { useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
 const App: React.FC = () => {
-  const [
-    selectedUserId,
-    // setSelectedUserId,
-  ] = useState(0);
+  const [selectedUserId, setSelectedUserId] = useState(0);
 
   return (
     <div className="App">
       <div className="App__sidebar">
-        <TodoList />
+        <TodoList selectUserId={setSelectedUserId} />
       </div>
 
       <div className="App__content">
         <div className="App__content-container">
           {selectedUserId ? (
-            <CurrentUser />
+            <CurrentUser
+              userId={selectedUserId}
+              setSelectedUserId={setSelectedUserId}
```

Comment:
It will be better, if you create callback and pass it to component CurrentUser instead state 'setSelectedUserId', because passing state is not a good idea

Reply:
Pay attention to @SavMary comments